usr.sbin/vmd/virtio.c

```diff
@@ -1898,16 +1892,16 @@ vionet_restore(FILE *fp, struct vm_create_params *vcp, int *child_taps) {
 
 void 
 vioblk_restore(FILE *fp, struct vm_create_params *vcp, int *child_disks) {
-	int i, ret;
+	unsigned int i;
```

Comment:
This should `uint8_t`
https://github.com/pdvyas/openbsd-src/pull/2/files#diff-ae0401923e6f6b919c7abaa89f579caeR1604